### ISSUE TITLE
Publish the chart using github pages

### ISSUE THREAD

**Issue by egabancho** [inveniosoftware/helm-invenio#133]
It'd be nice to pull this chart directly without cloning the repository. It seems that using GitHub pages is a sensible solution, even document on helm's docs: https://helm.sh/docs/howto/chart_releaser_action/

I know we will probably make breaking changes, always for the better, but that's what version [`0.x.y`](https://semver.org/#spec-item-4) is for, although we will try to be nice.

cc-ing @lindhe

**Comment by lindhe** [comment]
I agree 100%! Unless we publish the Helm chart, I don't understand why it exists. The whole point of Helm is to be a package manager, but our package has not been published so people cannot install it (easily).

> I know we will probably make breaking changes, always for the better, but that's what version [`0.x.y`](https://semver.org/#spec-item-4) is for, although we will try to be nice.

OMG yes!! 😅 Even with the chart being unpublished, it's a pain in the butt that we never increment the chart version number. It makes it very hard to keep track of which features are present in an installed release or not. And to publish it, we pretty much must start incrementing the version number on every change.

I've been wanting this since I first started working on the Helm chart, so I'm all with you. Just as you say, major version 0 specifically allows breaking changes so I see on reason to not increment it regularly. _Publishing the chart does not mean it's a stable release._

I think we need to have @slint and/or @ntarocco on board with us to approach this issue, since it primarily concerns infrastructure and release process rather than code changes.

**Comment by egabancho** [comment]
There is already a ["version" ](https://github.com/inveniosoftware/helm-invenio/blob/master/charts/invenio/Chart.yaml#L15). We can use that as the first one or change it to `0.1.0`. I don't have a strong preference here.

Do we like the idea of publishing the chart using GitHub pages?

**Comment by lindhe** [comment]
There is, but it has not been bumped in ages. It should be bumped on every chart change. It must be, if we shall publish the chart.

I think it sounds good, but I have no authority over the inveniosoftware organization's GitHub, so my vote does not count. 🙃

**Comment by slint** [comment]
IMHO, depends how we feel about the charts state. If we keep the `0.x.y` convention where every minor can be breaking, then I think we also make our lives a bit easier in terms of communicating the "official" release of the charts and have some leeway. In that case, bumping to v0.3.0, and setting up a GH-pages publishing sounds good to me.

If you're using this in production already, v1.0.0 is also fine :)

**Comment by lindhe** [comment]
That's great! I think we can keep it at major version zero for now. Just publishing it and start to use the version number would be a huge improvement from the current situation.

**Comment by egabancho** [comment]
```
 $ helm repo add invenio https://inveniosoftware.github.io/helm-invenio
"invenio" has been added to your repositories

$  helm search repo invenio
NAME            CHART VERSION   APP VERSION             DESCRIPTION
invenio/invenio 0.3.0           v12.0.0-beta.1.3        Turn-key research data management platform.
```
